nubilanding/src/components/footer/styles.module.scss

```
@@ -0,0 +1,126 @@
+.contenedor {
+  background-color: #6a6a6a;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  width: 100%;
+  padding-bottom: 20px;
+}
+
+.contenedorLogoNubi {
```

```suggestion
.contenedorLogoNubi {
```
poner todo junto lo que tenga que ver con posicionar al elemento-clase